apps/web/src/screens/HomePage/sections/Events/index.tsx

```diff
@@ -145,38 +171,60 @@ const CalendarLink = styled.a`
 `;
 
 const FILTER_TYPES: string[] = [
-  "ACM Comp",
-  "ACM Data",
-  "ACM Game",
-  "ACM General",
-  "ACM Hack",
-  "ACM Sec",
-  "ACM-W"
+  "Competition",
+  "Data",
+  "Game",
+  "General",
+  "Hack",
+  "Security",
+  "Women"
 ];
 
 const DEFAULT_EVENTS_TO_DISPLAY: number = 3;
```

Comment:
outside of the scope of your PR, but is this something we want to put in the config?